modules/custom_operations/examples/grid_sample/grid_sample.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class GridSample(torch.autograd.Function):
+    @staticmethod
+    def symbolic(g, x, grid):
+        return g.op('GridSample', x, grid)
+
+    @staticmethod
+    def forward(self, x, grid):
+        return F.grid_sample(x, grid, 'bilinear', 'zeros', True)
```

Comment:
We have native support for GridSample since OpenVINO 2022.2, and ONNX opset 16. Is there still a reason to keep this here, perhaps for legacy reasons?

Reply:
I got an accuracy error when running GridSample tests without the extension